pkg/kubectl/cmd/rollout/rollout_status.go

```diff
@@ -110,7 +114,8 @@ func RunStatus(f cmdutil.Factory, cmd *cobra.Command, out io.Writer, args []stri
 	if err != nil {
 		return err
 	}
-	rv, err := mapping.MetadataAccessor.ResourceVersion(obj)
```

Comment:
Are we losing watching from rv?

Reply:
Without digging into the resource builder code, I would hope/expect this to be the RV of either the list (if doing a list) or the single object (if doing a get). I assume the reason for capturing the RV was to be able to start a watch from the same RV instead of getting all the revisions currently available (anything that hasn't yet been collapsed into an etcd snapshot).

@liggitt any thoughts here?